documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+## Restricted Items
+1. Network ranges and IP addresses <br>
+The organisation currently have these classed as _sensitive_
```

Comment:
We care about secret information, such as passwords, secret access keys, QR codes (god forbid anyone would be daft enough to post those on git), etc. I think we need to be clearer on this.